fcrepo-http-api/src/main/resources/views/common-node-actions.vsl

```diff
@@ -50,13 +50,24 @@
     <input type="text" id="new_id" placeholder="(auto-generated identifier)" name="slug"  class="form-control"/>
         </div>
 
+    <div class="form-group" id="turtle_payload_container">
+    	<label for="rdf"  class="control-label">
+	        RDF (turtle)
+	    </label>
+		<textarea rows="10" id="turtle_payload" name="turtle" class="form-control" 
+		placeholder="@prefix pcdm: <http://pcdm.org/models#> 
+<> a pcdm:Object . ">
+	    </textarea>
```

Comment:
And given the whitespace preceding the `</textarea>` element, the placeholder text never actually appears (unless one removes the whitespace)